### PR TITLE
Change the adding cert secret from command to policy

### PR DESCRIPTION
**Pull request by yiraeChristineKim**
This will fix https://redhat-internal.slack.com/archives/CVAB9H3DE/p1712678150872979 issue

### CODE REVIEW

**Comment by dhaiducek** [comment]
Found it: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/stolostron_governance-policy-framework/782/pull-ci-stolostron-governance-policy-framework-main-test-e2e-grc-framework/1778053794530070528

```
Ran 176 of 182 Specs in 1744.791 seconds
SUCCESS! -- 176 Passed | 0 Failed | 0 Pending | 6 Skipped
PASS
```

**Review by dhaiducek — CHANGES_REQUESTED** [review]

File: test/integration/compliance_history_test.go [hunk 1]
```diff
@@ -661,6 +666,7 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
 			len(clusters)*expectedEvents))
 		Eventually(func(g Gomega) {
 			for _, cluster := range clusters {
+				By("Verifying cluster: " + cluster)
```
[the remark this answers is not in the document]
Comment (yiraeChristineKim) [reply test/integration/compliance_history_test.go:669]: Let's remove this. I don't think this is necessary

**Review by mprahl — DISMISSED** [review]

**Review by mprahl — APPROVED** [review]

**Pull request merged**